packages/cbioportal-utils/src/data/RemoteDataUtils.ts

```diff
@@ -3,7 +3,7 @@ import _ from 'lodash';
 import { RemoteDataStatus } from '../model/RemoteData';
 
 export function getRemoteDataGroupStatus(
```

Comment:
Apparently this was the duplicate of `getMobxPromiseGroupStatus` in `react-mutation-mapper`, now there is only one function in `cbioportal-utils`.